draft-ietf-tls-tls13.md

```diff
@@ -1553,8 +1553,12 @@ Ephemeral Secret (ES): A secret which is derived from fresh (EC)DHE
 Static Secret (SS): A secret which may be derived from static or
    semi-static keying material, such as a pre-shared key or the
    server's semi-static (EC)DH share.
+   
+Master Secret (SS): A secret derived from both the static 
```

Comment:
I'm actually thinking that the abbreviation would be useful.  The inconsistency is a little troublesome in a lot of places (tables and charts primarily).